src/Web/UIEvent/InputEvent.purs

```diff
@@ -23,3 +26,9 @@ toEvent = unsafeCoerce
 foreign import data_ :: InputEvent -> String
 
 foreign import isComposing :: InputEvent -> Boolean
+
+foreign import inputType_ :: InputEvent -> String
+
+inputType :: InputEvent -> Maybe InputType
+inputType = toEnumInputType <<< inputType_
```

Comment:
Instead of having this as a `Maybe` I'd suggest including an `Other String` constructor in `InputType` that will collect anything that falls through.